src/UserInterface/Core/MainWindow.h

```diff
@@ -94,6 +94,11 @@ class MainWindow : public QMainWindow {
 	 */
 	void showMesh();
 
+	/**
+	 * @brief  When mesh view is disabled -> add Model geometry actor to renderer
+	 */
+	void showGeometry();
```

Comment:
As I am developing the QIVtkRepresentationView class, I would like everything related to the display to be handled there, but it is not ready yet, so for the time being let's leave it this way